styleSheet.css

```diff
@@ -5,12 +5,12 @@ ul {
 #navBar{
   position: fixed;
   width: 100%;
-  height: 5%;
+  height: 5.5%;
   top: 0;
   background-color: #333;
 }
 
-#navBar>div{
+#navBar > nav{
```

Comment:
why do we need to wrap the nav inside a div called `#navBar`?

Reply:
Well, I am using it so I can center the nav's text/links in the middle of the page

Reply:
Yeah already fixed no more  nav inside a div